app/controllers/api/v1/emotions_controller.rb

```diff
@@ -73,4 +77,12 @@ def emotion_params
   def time_period
     @time_period ||= TimePeriod.find_or_create_time_period
   end
+
+  def list_shoutouts_to(user)
+    list_ids_shot_outs = RecipientShoutout.where(user_id: user.id)
```

Comment:
This is not a good variable name since it will be a list of objects rather than a list of only ids.

Reply:
Thanks I decided to renamed it to the 'shoutouts_to_user'